### ISSUE TITLE
Mod

### ISSUE THREAD

**Issue by KeinNiemand** [Krafpy/KSP-MGA-Planner#6]
An MGA planer would be more usefull as a mod.

**Comment by nmisyats** [comment]
I agree. I plan to work on a mod version of my MGA planner when I get the time for it.
